apps/builder/app/shared/nano-states/nano-states.ts

```diff
@@ -233,8 +233,13 @@ export const selectedInstanceIntanceToTagStore = atom<
   undefined | Map<Instance["id"], HtmlTags>
 >();
 
-/** Whether or the selected instance is rendered on canvas */
-export const selectedInstanceIsRenderedStore = atom<boolean>(false);
+/**
+ * Whether or not the selected instance is rendered on canvas:
```

Comment:
This sentence is confusing, maybe the variable name is confusing with this semantics?

Kinda can't use "is" if its not boolean, you need to change the name

Reply:
seems like better name here would be: "selectedInstanceRenderState": "mounted" | "unmounted" | "something else" ?

Reply:
Ok